.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,31 @@
+trigger:
+- master
+
+pr:
+- master
+
+resources:
+  repositories:
+  - repository: azure_template
+    type: github
+    name: mdolab/.github
+    endpoint: mdolab
+
+pool:
+    vmImage: 'ubuntu-latest'
+
+extends:
+  template: azure/azure_template.yaml@azure_template
+  parameters:
+    REPO_NAME: adflow
+    IGNORE_STYLE: true
+    COMPLEX: true
+    GCC_CONFIG: config/defaults/config.LINUX_GFORTRAN.mk
+    INTEL_CONFIG: config/defaults/config.LINUX_INTEL.mk
+    INPUT_FILES: true
+    STEPS_MAKE_REAL: make
+    STEPS_INSTALL_REAL: pip install .
+    STEPS_TEST_REAL: testflo -v .
```

Comment:
Note that Travis has `-n 1` appended to the `testflo` commands, we probably want the same here.

Reply:
Got it -- I'll add this in.